commercetools-models/src/main/java/io/sphere/sdk/shoppinglists/LineItem.java

```diff
@@ -0,0 +1,54 @@
+package io.sphere.sdk.shoppinglists;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.annotations.ResourceValue;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.products.ProductVariant;
+import io.sphere.sdk.producttypes.ProductType;
+import io.sphere.sdk.types.CustomFields;
+import io.sphere.sdk.types.TypeDraft;
+
+import javax.annotation.Nullable;
+import java.time.ZonedDateTime;
+
+@JsonDeserialize(as = LineItemImpl.class)
+@ResourceValue
+public interface LineItem {
```

Comment:
Can you write here some documentation like the one that cart's line item has? 

Something like:
```
A line item is a reference to a product variant in a product in its current version.

A LineItem can have {@link io.sphere.sdk.types.Custom custom fields}.

 @see io.sphere.sdk.shoppinglists.commands.updateactions.AddLineItem
 @see io.sphere.sdk.shoppinglists.commands.updateactions.RemoveLineItem
 @see io.sphere.sdk.shoppinglists.commands.updateactions.ChangeLineItemQuantity
 @see ShoppingList#getLineItems()
```

I know it's difficult to keep up to date, so we can discuss it later too.